channel/state/state.go

```diff
@@ -69,21 +69,27 @@ var address, _ = abi.NewType("address", "address", nil)
 
 // ChannelId computes and returns the channel id corresponding to the State,
 // and an error if the id is an external destination.
+// Up to hash collisions, ChannelId distinguishes channels that have different FixedPart
```

Comment:
```suggestion
//
// Up to hash collisions, ChannelId distinguishes channels that have different FixedPart
```

Paragraph break requires the empty line. If you don't intend a <p> here, disregard.

Reply:
ea6936d72f09af3b0dd1eda272fe2141c81499de